controllers/notifyController.php

```diff
@@ -54,24 +53,31 @@ class WcPagantisNotify extends WcPagantisGateway
     /** @var $string */
     protected $product;
 
+    /** @var $string */
+    protected $verificationToken = null;
+
     /**
      * Validation vs PagantisClient
      *
      * @return JsonExceptionResponse|JsonSuccessResponse
      * @throws ConcurrencyException
      */
-    public function processInformation()
+    public function processNotification()
     {
         try {
-            require_once(__ROOT__.'/vendor/autoload.php');
+            require_once(__ROOT__ . '/vendor/autoload.php');
             try {
                 if ($_SERVER['REQUEST_METHOD'] == 'GET' && $_GET['origin'] == 'notification') {
                     return $this->buildResponse();
                 }
+
+                $this->setWoocommerceOrderId();
+                $this->setVerificationToken();
+                $this->verifyOrderConformity();
                 $this->checkConcurrency();
-                $this->getProductType();
+                $this->setPaymentProductType();
                 $this->getMerchantOrder();
-                $this->getPagantisOrderId();
+                $this->setPagantisOrderId();
```

Comment:
Please keep the conventional naming: https://digitalorigin.atlassian.net/wiki/spaces/MERCHD/pages/599752794/Order+Confirmation+Flow

Reply:
done